BraveRewards/Ledger/Data/Model.xcdatamodeld/Model.xcdatamodel/contents

```diff
@@ -8,7 +8,7 @@
         <attribute name="score" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES" syncable="YES"/>
         <attribute name="visits" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="YES" syncable="YES"/>
         <attribute name="weight" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES" syncable="YES"/>
-        <relationship name="publisher" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="PublisherInfo" syncable="YES"/>
+        <relationship name="publisher" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="PublisherInfo" inverseName="activities" inverseEntity="PublisherInfo" syncable="YES"/>
```

Comment:
Do we need the cascade deletion rule on the original relationships?

Reply:
I don't think so, but let me confirm and write a test or two